util.sh

```diff
@@ -49,3 +49,6 @@ OCI_VERSION="${PODMAN_VERSION%.*}"
 FULL_IMAGE_NAME="${REPO}/${OCI_NAME}:${OCI_VERSION}"
 
 export FULL_IMAGE_NAME_ARCH="$FULL_IMAGE_NAME-${ARCH_TO_IMAGE_ARCH[$CPU_ARCH]}"
+
+FEDORA_LATEST_STABLE=$(resolve-fedora-aliases --output-type branch fedora-latest-stable)
```

Comment:
Sorry I just noticed this now, this is not necessarily the same fedora version as the coreos version we derive from.
i.e. the core stable steam will take a bit longer to update so we likely download the wrong version once f42 is released. And I doubt we want to install f42 packages on a f41 base?

We likely need a way to keep this and the version from the coreos image in sync. Especially as I assume we need from time to time to pin coreos to an older version due to new regressions like we have right now.

So should we pin this manually to 41 and update once coreos is updated? I am not sure what the best way to keep both in sync is. I guess overall it would be good if we notice if the fedora base was upgraded in the coreos stream we consume.

cc @baude

Reply:
ack, let me see if I can try to base it automatically on the coreos version we fetch.